.travis.yml

```diff
@@ -1,24 +1,35 @@
 language: python
 sudo: false
 python:
-  - "2.6"
-  - "2.7"
-install:
-  - pip install python-coveralls
+  - 2.6
+  - 2.7
+
 env:
-  global:
-    - "PIP_DOWNLOAD_CACHE=$HOME/.pip-cache"
```

Comment:
I assume you removed it because that has been a default while in pip for a while now?